docker-compose.yml

```diff
@@ -3,16 +3,19 @@ db:
   environment:
     POSTGRES_USER: postgres
     POSTGRES_PASSWORD: postgres
+cache:
+  image: library/memcached
 web:
   image: kinto/kinto-server
   links:
    - db
+   - cache
   ports:
    - "8888:8888"
   environment:
-    KINTO_CACHE_BACKEND: kinto.core.cache.postgresql
-    KINTO_CACHE_URL: postgres://postgres:postgres@db/postgres
+    KINTO_CACHE_BACKEND: kinto.core.cache.memcached
+    KINTO_CACHE_HOSTS: cache:11211 cache:11212
     KINTO_STORAGE_BACKEND: kinto.core.storage.postgresql
     KINTO_STORAGE_URL: postgres://postgres:postgres@db/postgres
     KINTO_PERMISSION_BACKEND: kinto.core.permission.postgresql
-    KINTO_PERMISSION_URL: postgres://postgres:postgres@db/postgres
+    KINTO_PERMISSION_URL: postgres://postgres:postgres@db/postgres                                                             
```

Comment:
lot of spaces at the end.

Reply:
@emamurho please fix this and we're good to go!

@Natim did you try the changes locally?

Reply:
Yes I did, it seems to works :)